portfolio/src/main/java/com/google/sps/data/Video.java

```diff
@@ -33,6 +34,16 @@ public Video(String videoId, String title, String description, String thumbnailU
         this.captions = null;
     }
 
+    public Video(String videoId, String title, String description, String thumbnailUrl, String captions, boolean isPublic) 
+      throws MalformedURLException{
+        this.videoId = videoId;
+        this.title = title;
+        this.description = description;
+        this.thumbnailUrl = new URL(thumbnailUrl);
+        this.isPublic = isPublic;
+        this.captions = captions;
+    }
+
     // Create video from videoId.
```

Comment:
We can delete this now.

Reply:
Delete what? The "Create video from videoId" function?

Reply:
Yes, we can delete it or have it just call your function.